Havit.Blazor.Components.Web.Bootstrap/Tags/Internal/HxInputTagsInternal.razor

```diff
@@ -1,6 +1,6 @@
 @namespace Havit.Blazor.Components.Web.Bootstrap.Internal
 
-<span class="@CssClassHelper.Combine((HasInputGroupsEffective ? "input-group" : null), InputSizeEffective.AsInputGroupCssClass(), InputGroupCssClass)">
+<span class="@CssClassHelper.Combine((HasInputGroupsEffective ? "input-group" : null), InputSizeEffective.AsInputGroupCssClass(), InputGroupCssClass, "form-floating")">
```

Comment:
I doubt we want the `form-floating` class to be there all the time....?

Reply:
@hakenr I agree. This is still a work-in-progress - the label shows correctly but doesn't animate and fill the input when there is no value entered.